src/tendencies/implicit/implicit_tendency.jl

```diff
@@ -16,39 +16,46 @@ import ClimaCore.Spaces as Spaces
 # the implicit tendency function. Since dt >= dtγ, we can safely use dt for now.
 function vertical_transport!(ᶜρcₜ, ᶠw, ᶜρ, ᶜρc, p, ::Val{:none})
     (; dt) = p.simulation
-    (; ᶜdivᵥ, ᶠinterp) = p.operators
-    @. ᶜρcₜ = -(ᶜdivᵥ(ᶠinterp(ᶜρc) * ᶠw))
+    (; ᶜdivᵥ, ᶠwinterp) = p.operators
+    J = Fields.local_geometry_field(axes(ᶜρ)).J
+    @. ᶜρcₜ = -(ᶜdivᵥ(ᶠwinterp(J, ᶜρc) * ᶠw))
 end
```

Comment:
why do we need these changes in this PR?

Reply:
We do not; once 1240 is merged, this will be rebased on the latest changes which should collapse some of the modified lines of code.